.github/workflows/push.yml

```diff
@@ -101,8 +118,23 @@ jobs:
             echo "Pull request does not have deploy-feature label, exiting..."
             exit 0
           fi
+          for pre_pattern in $(echo $PRE_RELEASE_PATTERN | sed "s/,/ /g")
+            do
+                echo "Checking branch against pre_release_pattern '$pre_pattern'"
+                echo "Check if this is a pre-release and if it should generate a feature-deploy"
+                if [[ "$GIT_BRANCH" =~ $pre_pattern ]]; then
+                  echo "'$GIT_BRANCH' matches 'pre_$pattern', continuing with build"  
+                  echo "::set-output name=PRE_CHECK::feature-deploy"
+                  echo "::set-output name=PRE_RELEASE::true"
+                  exit 0
+                fi
+            done
```

Comment:
Can we get rid of this duplication?

Reply:
Not sure. It was your idea :D 
One of them is checking when we create a branch and the other is checking on any other event, while skipping PR